assets/src/scss/pages/post/_post.scss

```diff
@@ -1,96 +1,90 @@
-.single-post,
-.single-attachment,
-.single-idea {
-  @include large-and-up {
-    .narrow-container {
-      margin-left: auto;
-      margin-right: auto;
-      max-width: 776px;
-      padding: 0;
-    }
+@include large-and-up {
+  .narrow-container {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 776px;
+    padding: 0;
   }
+}
 
-  @include x-large-and-up {
-    .narrow-container {
-      max-width: 736px;
-    }
+@include x-large-and-up {
+  .narrow-container {
+    max-width: 736px;
   }
+}
 
-  .skewed-overlay {
-    mask-image: none;
-  }
+.skewed-overlay {
+  mask-image: none;
+}
 
-  .page-header {
-    .page-header-title {
-      color: $grey-80;
-      hyphens: none;
-      max-width: none;
-      margin-bottom: 8px;
-      line-height: 1.2;
-    }
+.page-header-title {
+  color: $grey-80;
+  hyphens: none;
+  max-width: none;
+  margin-bottom: 8px;
+  line-height: 1.2;
+}
 
-    .single-post-meta {
-      color: $grey-40;
-      font-size: 14px;
-      font-family: $roboto;
-      border-bottom: 1px solid transparentize($grey-20, 0.5);
-      padding-bottom: 16px;
-      line-height: 24px;
-      margin-bottom: 16px;
-      display: flex;
-      gap: 8px;
-      align-items: end;
+.single-post-meta {
+  color: $grey-40;
+  font-size: 14px;
+  font-family: $roboto;
+  border-bottom: 1px solid transparentize($grey-20, 0.5);
+  padding-bottom: 16px;
+  line-height: 24px;
+  margin-bottom: 16px;
+  display: flex;
+  gap: 8px;
+  align-items: end;
 
-      a {
-        color: inherit;
-      }
+  a {
+    color: inherit;
+  }
+}
 
-      .single-post-meta-bullet {
-        margin: 0 8px;
-      }
+.single-post-meta-bullet {
+  margin: 0 8px;
+}
 
-      .author-pic {
-        border-radius: 50%;
-        object-fit: cover;
-        max-height: 50px;
-      }
+.author-pic {
+  border-radius: 50%;
+  object-fit: cover;
+  max-height: 50px;
+}
 
-      .single-post-author {
-        display: block;
-        margin: 0;
-        font-weight: 500;
-      }
+.single-post-author {
+  display: block;
+  margin: 0;
+  font-weight: 500;
+}
 
-      .single-post-time,
-      .comment-link {
-        display: inline-block;
-        margin: 0;
-      }
+.single-post-time,
+.comment-link {
+  display: inline-block;
+  margin: 0;
+}
 
-      .comment-link {
-        font-weight: 500;
+.comment-link {
+  font-weight: 500;
 
-        img {
-          width: 17px;
-        }
+  img {
+    width: 17px;
+  }
 
-        svg {
-          top: -2px;
-        }
-      }
-    }
+  svg {
+    top: -2px;
   }
+}
 
-  .post-articles-block .articles-block {
```

Comment:
Can we check in a follow up? The `single.twig` template is [overridden by a few child themes](https://github.com/search?q=org%3Agreenpeace+post-articles-block&type=code
). 
Also the specificity change maybe affects[ CSS that targets `.articles` block only](https://github.com/search?q=org%3Agreenpeace+articles-block&type=code).

Reply:
Yes, I'm fine with removing this in a follow-up! However the specificity changes should not affect existing Articles blocks right? Since the Articles block is currently not allowed in posts, and this CSS is only for posts 🤔

Reply:
It should be fine in our base CSS, and probably also in the child themes, but it's harder to tell without looking at each in detail.